packages/core/src/providers/chainState/provider.tsx

```diff
@@ -52,6 +53,7 @@ export function ChainStateProvider({ children, multicallAddresses }: Props) {
         })
         .catch((error) => {
           console.error(error)
+          reportError(error)
```

Comment:
Shall we move `console.error()` to `reportError`?

Reply:
Let's try to, if it will spam too much, we can drop it